src/WebJobs.Extensions.OpenAI.CosmosDBNoSqlSearch/WebJobs.Extensions.OpenAI.CosmosDBNoSqlSearch.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
please add this project to the solution as well

Reply:
I am not sure what needs to be done for this? The project is already there in the OpenAI-Extension.sln.

Reply:
Added